app/oauth.go

```diff
@@ -181,7 +184,7 @@ func (a *App) AllowOAuthAppAccessToUser(userId string, authRequest *model.Author
 		Value:    authRequest.Scope,
 	}
 
-	if result := <-a.Srv.Store.Preference().Save(&model.Preferences{authorizedApp}); result.Err != nil {
+	if result = <-a.Srv.Store.Preference().Save(&model.Preferences{authorizedApp}); result.Err != nil {
```

Comment:
Why is this if-error-check conditional not being removed like the rest of them?

Reply:
Because we only check if an error is returned. See https://developers.mattermost.com/blog/idiomatic-error-handling/#is-it-ok-to-still-declare-variables-inline-to-a-conditional